apps/namadillo/src/App/Masp/ShieldedFungibleTable.tsx

```diff
@@ -126,7 +126,7 @@ export const ShieldedFungibleTable = ({
             outlineColor="white"
             className="w-fit ml-auto mr-10"
             onClick={() =>
-              navigate(`${routes.unshield}?${params.asset}=${address}`)
+              navigate(`${routes.maspUnshield}?${params.asset}=${address}`)
```

Comment:
Yeah so maybe for simplicty this could navigate to transfer route with filled, source,target and token. Btw I've broken this a bit with last commit cause I changed token address to token symbol in the URL

Reply:
https://github.com/user-attachments/assets/f0363053-dd8e-4518-aa85-4ff0136b068a


Fixed it for shield/unshield 👍